styles.css

```diff
@@ -0,0 +1,67 @@
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+    font-family: 'Inter', sans-serif;
+}
+.cta {
+    background: #2EC5CE;
+    display: flex;
+    justify-content: space-around;
+    margin: auto;
+
+}
+.cta-left {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    justify-content: center;
+    margin: 70px 75px 70px 165px;
+    gap: 48px;
+    
+}
+.cta-left-top {
+    width: 540px;
```

Comment:
let's make it max-width instead of width(as it will make the container fixed at a given width ie. 540px when screen size gets reduced)